tools/apksize/apksize.gradle

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import com.google.firebase.gradle.GenerateMeasurementsTask
+import com.google.firebase.gradle.UploadMeasurementsTask
+
+// Linting needs to be disabled as none of these apps are real or intended to conform to
```

Comment:
While true, enabling linting can bring this app closer to being a "real" app.
It might be worth taking this on as tech debt.

Reply:
I agree. This was in the original code, and I'd like to re-enable lint in the future.

Reply:
Can we add a Github issue to enable this?